lib/order.rb

```diff
@@ -0,0 +1,90 @@
+require 'twilio-ruby'
+
+class Order
+
+    attr_reader :menu, :items
+
+    def initialize(menu)
+        @menu = menu
+        @items = []
+        @items_detail
```

Comment:
I think this should be defined as to what is it here and not in the method if I am correct. I think it is given an empty hash in summarise order.